### ISSUE TITLE
I recommand you to catch error 

### ISSUE THREAD

**Issue by ousamabenyounes** [karim1706/My_Animes_Library#1]
Searching on google, i found this sample : 

`[](https://www.tabnine.com/code/javascript/functions/axios/Promise/catch)questionnaire_web/src/store/questionnaireTemplate.js/undefined/removeTemplateById_2
removeTemplateById({commit, state}, {templateId, index}) {
  return new Promise((resolve, reject) => {
   axios.patch(`${bus.host}/questionnaireTemplate/${templateId}/remove`)
    .then(function ({data}) {
     commit('removeTemplateById', {index});
     resolve(null, data);
    })
    .catch(function (error) {
     resolve(error);
    });

  });
 }`
found here => ["https://www.tabnine.com/code/javascript/functions/axios/Promise/catch"]( https://www.tabnine.com/code/javascript/functions/axios/Promise/catch)

so if the api is down (they didn't pay their aws instances :)..) you'll inform the user that something is wrong...sorry :)
good luck

https://github.com/karim1706/My_Animes_Library/blob/be73acc9db0c8fc9469e941d4691944ac8cc0374/js/apiCall.js#L32

**Comment by karim1706** [comment]
You're right ! It seems that I have to migrate to their V4 API's version. And yes I forgot to catch the error ! I'll check that ! Thank you very much Ousama, I really appreciate your help !
